@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -897,6 +930,9 @@ export class VisibleTileSet {
                     tile.offset
                 );
                 tile.levelOffset = 0;
+                if (tile.backgroundPlane !== undefined) {
```

Comment:
Why is this needed?

Reply:
I think a left over from a previous attempt, but not sure, will remove.